db/seeds.rb

```diff
@@ -68,14 +68,14 @@ def log(msg)
 alice = Receiver.create!(
   receiver_sso_id: Faker::Number.number(digits = 8),
   email: 'alice@aol.com',
-  mobile_number: Faker::PhoneNumber.cell_phone,
+  mobile_number: ENV['ALICES_MOBILE_NUMBER'],
```

Comment:
why do you have this as an env var?

Reply:
Because I don't want the actual cell phone that I use for testing to be stored in git.